README.md

```diff
@@ -140,6 +140,21 @@ Returns group metadata for a given group:
 
 Lists all the authors (feedIds) who you know are part of the group with id `groupId`
 
+### `ssb.tribes.subtribes.create(groupId, opts, cb)`
```

Comment:
we have `group.create` I think this should also then be singular : `subtribe.create`

Reply:
I think it should be singular `ssb.tribes.subtribe.create()`

Reply:
Okay :)